tests/views/api_tests.py

```diff
@@ -74,24 +74,24 @@ def setUpClass(cls):
         test_pkg_path = os.path.join(test_settings.TEST_ROOT, "fixtures", "testing_prj", "testing_prj", "pkg")
         management.call_command("packages", operation="load_package", source=test_pkg_path, yes=True)
 
-    def test_api_base_view(self):
```

Comment:
I think the mobilesurvey endpoint could just be swapped out for another endpoint.  The test passed for me using `api_node_value`.


```
    def test_api_base_view(self):
        """
        Test that our custom header parameters get pushed on to the GET QueryDict

        """
        factory = RequestFactory(HTTP_X_ARCHES_VER="2.1")
        view = APIBase.as_view()

        request = factory.get(reverse("api_node_value", kwargs={}), {"ver": "2.0"})
        request.user = None
        response = view(request)
        self.assertEqual(request.GET.get("ver"), "2.0")

        request = factory.get(reverse("api_node_value"), kwargs={})
        request.user = None
        response = view(request)
        self.assertEqual(request.GET.get("ver"), "2.1")
        ```